drivers/dma/dma_intel_adsp_gpdma.c

```diff
@@ -134,6 +136,7 @@ static int intel_adsp_gpdma_start(const struct device *dev, uint32_t channel)
 	if (ret != 0) {
 		intel_adsp_gpdma_llp_disable(dev, channel);
 	}
+	pm_device_runtime_get(dev);
```

Comment:
if dw_dma_start returns error, do you still want to call pm_device_runtime_get() ?